Bio/Align/__init__.py

```diff
@@ -926,6 +928,429 @@ def sort(self, key=None, reverse=False):
+    subsequences that align the best.
+
+    To perform a pairwise sequence alignment, first create a PairwiseAligner
+    object.  This object stores the match and mismatch scores, as well as the
+    gap scores.  Typically, match scores are positive, while mismatch scores
+    and gap scores are negative or zero.  By default, the match score is 1,
+    and the mismatch and gap scores are zero.  Based on the values of the gap
+    scores, a PairwiseAligner object automatically chooses the appropriate
+    alignment algorithm (the Needleman-Wunsch, Smith-Waterman, Gotoh, or
+    Waterman-Smith-Beyer global or local alignment algorithm).
+
+    Calling the "score" method on the aligner with two sequences as arguments
+    will calculate the alignment score between the two sequences.
+    Calling the "align" method on the aligner with two sequences as arguments
+    will return a generator yielding the alignments between the two
+    sequences.
+
+    Some examples:
+    >>> from Bio import Align
+    >>> aligner = Align.PairwiseAligner()
+    >>> alignments = aligner.align("ACCGT", "ACG")
+    >>> for alignment in sorted(alignments):
+    ...     print("Score = %.1f:" % alignment.score)
+    ...     print(alignment)
+    Score = 3.0:
+    ACCGT
+    | || 
+    A-CG-
+    <BLANKLINE>
+    Score = 3.0:
+    ACCGT
+    || | 
+    AC-G-
+    <BLANKLINE>
+
+    Specify the aligner mode as local to generate local alignments:
+    >>> aligner.mode = 'local'
+    >>> alignments = aligner.align("ACCGT", "ACG")
+    >>> for alignment in sorted(alignments):
+    ...     print("Score = %.1f:" % alignment.score)
+    ...     print(alignment)
+    Score = 3.0:
+    ACCGT
+    | || 
+    A-CG 
+    <BLANKLINE>
+    Score = 3.0:
+    ACCGT
+    || | 
+    AC-G 
+    <BLANKLINE>
+
+    Do a global alignment.  Identical characters are given 2 points,
+    1 point is deducted for each non-identical character.
+    >>> aligner.mode = 'global'
+    >>> aligner.match = 2
+    >>> aligner.mismatch = -1
+    >>> for alignment in aligner.align("ACCGT", "ACG"):
+    ...     print("Score = %.1f:" % alignment.score)
+    ...     print(alignment)
+    Score = 6.0:
+    ACCGT
+    || | 
+    AC-G-
+    <BLANKLINE>
+    Score = 6.0:
+    ACCGT
+    | || 
+    A-CG-
+    <BLANKLINE>
+
+    Same as above, except now 0.5 points are deducted when opening a
+    gap, and 0.1 points are deducted when extending it.
+    >>> aligner.open_gap_score = -0.5
+    >>> aligner.extend_gap_score = -0.1
+    >>> aligner.target_end_gap_score = 0.0
+    >>> aligner.query_end_gap_score = 0.0
+    >>> for alignment in aligner.align("ACCGT", "ACG"):
+    ...     print("Score = %.1f:" % alignment.score)
+    ...     print(alignment)
+    Score = 5.5:
+    ACCGT
+    | || 
+    A-CG-
+    <BLANKLINE>
+    Score = 5.5:
+    ACCGT
+    || | 
+    AC-G-
+    <BLANKLINE>
+
+    The alignment function can also use known matrices already included in
+    Biopython ( Bio.SubsMat -> MatrixInfo ).
```

Comment:
You can probably remove the `` ( Bio.SubsMat -> MatrixInfo ).`` bit and just use a colon as the example shows this.

Reply:
OK